src/NServiceBus.AcceptanceTests/UnitOfWork/TransactionScope/When_used_with_default_transaction_mode.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace NServiceBus.AcceptanceTests.UnitOfWork
+{
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using Configuration.AdvanceExtensibility;
+    using EndpointTemplates;
+    using FakeTransport;
+    using NUnit.Framework;
+
+    public class When_used_with_default_transaction_mode : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_work()
+        {
+            var context = await Scenario.Define<Context>()
+                 .WithEndpoint<ScopeEndpoint>(b => b.CustomConfig(c =>
+                 {
+                     c.GetSettings().Set("FakeTransport.SupportedTransactionMode", TransportTransactionMode.ReceiveOnly);
```

Comment:
@andreasohlund we should probably raise an issue, since there are also users who would like to test their feature setups (e.g. maybe provide a `TestableFeatureSetupContext`)